frontend/src/Pages/TeamDebateRoom.tsx

```
@@ -0,0 +1,1685 @@
+              console.log(`Role selection from my team (${messageTeamId}): ${data.role}`);
+            } else {
+              // This role selection is from opponent team
+              setPeerRole(data.role as DebateRole);
+              console.log(`Role selection from opponent team (${messageTeamId}): ${data.role}`);
+            }
+          }
+          break;
+        case "countdownStart":
+          // Backend is starting countdown - show it to all users
+          const countdownValue = (data as any).countdown || 3;
+          console.log('✓✓✓ COUNTDOWN STARTED FROM BACKEND:', countdownValue);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)